web/cobrands/oxfordshire/base.scss

```diff
@@ -5,6 +5,17 @@
 @import "../sass/base";
 @import "../sass/report_list_pins";
 
+body {
+    background-color: $base_bg;
```

Comment:
Should we make this a variable for all cobrands? I see it could default to white, but a few would change it.